dms/views/directory.xml

```diff
@@ -0,0 +1,731 @@
+                                    <field name="perm_unlink" />
+                                </tree>
+                            </field>
+                        </page>
+                        <page name="mail_alias" string="Email Alias">
+                            <group>
+                                <group>
+                                    <field
+                                        name="alias_contact"
+                                        string="Accept Emails from"
+                                    />
+                                    <field name="alias_process" />
+                                </group>
+                                <group>
+                                    <label for="alias_name" string="Email Alias" />
+                                    <div>
+                                        <field name="alias_name" class="oe_inline" />
+                                        <span class="oe_read_only">@</span>
+                                        <span class="oe_edit_only"> @ </span>
+                                        <field
+                                            name="alias_domain"
+                                            placeholder="mail.catchall.domain"
+                                            class="oe_inline"
+                                            readonly="1"
+                                        />
+                                    </div>
+                                    <field
+                                        name="alias_id"
+                                        readonly="1"
+                                        required="0"
+                                        groups="base.group_no_one"
+                                        string="Alias Setting"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" />
+                    <field name="activity_ids" />
+                    <field name="message_ids" />
+                </div>
+            </form>
+        </field>
+    </record>
+    <record id="view_dms_directory_manager_form" model="ir.ui.view">
+        <field name="name">dms_directory.form</field>
+        <field name="model">dms.directory</field>
+        <field name="inherit_id" ref="view_dms_directory_form" />
+        <field name="arch" type="xml">
+            <field name="is_root_directory" position="after">
+                <field name="is_root_directory" groups="dms.group_dms_manager" />
+            </field>
+            <field name="storage_id" position="attributes">
+                <attribute name="options">{}</attribute>
+                <attribute name="groups">dms.group_dms_manager</attribute>
+            </field>
+        </field>
+    </record>
```

Comment:
```suggestion
```
This view should be deleted according to the previous comment.

Reply:
I think you must have forgotten to duplicate the field in the parent view; one with the `no_open` option and groups `!dms.group_dms_manager`, the other with the `groups dms.group_dms_manager` and without the option. If you do so you should be good.

Reply:
`directory.xml`
![image](https://github.com/OCA/dms/assets/25118938/b5283b37-5afd-4518-86be-c7b6f2193419)

No, I did not forget it.

Reply:
I added a new code suggestion below. Basically, if you duplicate the fields in the parent view, you don't need the override altogether. But the comments are a bit confused with all that is going on.

My tip with that is generally to address reviewer comments in specific commits, and squash things as necessary at the very last moment before merge, so that it's easy to keep track of everything during back and forth.

If you need I'll do another branch so that you can cherry-pick fixes to that.

Reply:
Done!